README.md

```diff
@@ -118,6 +118,10 @@ Open index.html using live server extension.
 - [ ] A complete website with navigation
 - [ ] Add more styles to make it more awesome
 
+👤 **Md Arafat Hossain**
+- GitHub: <a href="https://github.com/HossainAraf">HossainAraf </a>
+- LinkedIn: <a href="https://linkedin.com/in/md-arafat-hossain-111403275"> Md. Arafat Hossain </a>
+
```

Comment:
- [ ] It seems like while adding the *Future Feature* section you messed up with the `Authors` section of your `README` file and it looks terrible and can confuse someone very easily. Please always make sure that your *README* file is professional and that there is no redundancy in your document. Kindly fix your README file.

![Screenshot from 2023-09-01 14-34-08](https://github.com/misikir21/mama/assets/66476329/98bc2088-d110-43b2-abce-d2703530a237)